docs/deployment/caching/etags.md

```diff
@@ -0,0 +1,75 @@
+---
+myst:
+  html_meta:
+    "description": "ETags support for Plone caching"
+    "property=og:description": "ETags support for Plone caching"
+    "property=og:title": "ETags support for Plone"
+    "keywords": "Plone, deployment, automation, caching"
+---
+
+(caching-etags-label)=
+
+# ETags
+
+ETags are used in to determine whether pages need to be re-calculated or can be served from cache.
+An ETag generated by `plone.app.caching` is a string of tokens separated by pipe characters that hold values like a user ID, the current skin name, or a counter indicating how many objects have been added to the site.
```

Comment:
just found: "the current skin name" - is this nowadays a good example?

Reply:
This is still in `plone.app.caching`